front/src/Phaser/Game/GameScene.ts

```diff
@@ -37,12 +39,22 @@ export class GameScene extends Phaser.Scene {
     startY = 32; // 1 case
     circleTexture: CanvasTexture;
     initPosition: PositionInterface;
+    private playersPositionInterpolator = new PlayersPositionInterpolator();
 
     MapKey: string;
     MapUrlFile: string;
     RoomId: string;
     instance: string;
 
+    currentTick: number;
+    lastSentTick: number; // The last tick at which a position was sent.
+    lastMoveEventSent: HasMovedEvent = {
+        direction: '',
+        moving: false,
+        x: -1000,
```

Comment:
why `-1000` in the initial variable ?

Reply:
I need to initiate it to some random value (this avoids me to do a "null" check that is useful only for the first move) and I need to be sure that this will be overwritten when the user is making the first move (so that lastMoveEventSent is different from the next event). By putting x=-1000, I'm sure that lastMoveEventSent != next event sent.